content/en/docs/Concepts/olm-architecture/dependency-resolution/_index.md

```diff
@@ -510,8 +535,8 @@ OLM performs dependency resolution at the namespace scope. It is possible to get
 
 ## Backwards-Compatibility Notes
 
-`dependencies.yaml` is supported in `0.16.1+` versions of OLM.
+`dependencies.yaml` is supported in `0.16.1+` versions of OLM and `0.10.1+ versions of OPM.
```

Comment:
It looks like we're missing the closing backtick here.

```suggestion
`dependencies.yaml` is supported in `0.16.1+` versions of OLM and `0.10.1+` versions of OPM.
```

Reply:
Actually do we have any tagged releases that are prefixed with the `0.*` major version? I guess implicitly registry might still be pre-v1 but I'm not seeing anything https://github.com/operator-framework/operator-registry/tags that matches that tagging pattern.

Reply:
Fixed a couple of typos there.  Should have been `1.10.0+`.